tests/test_regex_sentence_generator.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+from aoptk.regex_sentence_generator import RegexSentenceGenerator
+from aoptk.sentence_generator import SentenceGenerator
+
+
+def test_can_create():
+    """Test that RegexSentenceGenerator can be instantiated."""
+    actual = RegexSentenceGenerator()
+    assert actual is not None
+
+
+def test_implements_interface():
+    """Test that RegexSentenceGenerator implements SentenceGenerator interface."""
+    assert issubclass(RegexSentenceGenerator, SentenceGenerator)
+
+
+def test_generate_sentences_not_empty():
+    """Test that generate_sentences method returns a non-empty result."""
+    actual = RegexSentenceGenerator().generate_sentences("")
+    assert actual is not None
+
+
+@pytest.fixture(
```

Comment:
You could also use the """ to write longer text sections, which would make these a bit prettier. Another option if you want to test longer sections would be to write a few sentences into a text file and then load it - but these are details. I'm just saying it to highlight alternatives but there is no need to change something here at the moment.